wikimedia/wikipedia-ios
# Update `MobileWikiAppiOSUserHistory` to add `inbox_count` and `device_level_enabled` values

**Phabricator:** https://phabricator.wikimedia.org/T297752

### Notes
* This PR updates the `MobileWikiAppiOSUserHistory` schema to be able to collect data from our notifications, We're adding two more properties - `inbox_count` that contains the amount of notifications a user has as an Integer, and `device_level_enabled` which sends the  current `UNAuthorizationStatus` as a String.

### Test Steps
1. Launch the app
2. In Settings, go to Notifications and enable them
3. If possible, do some actions that generate notifications (so we can see the count go up and update accordingly)
4. Check if `MobileWikiAppiOSUserHistory` has the two new properties added, and if they reflect currently the user's inbox count and notifications authorization status.
